manifests/config/project.pp

```diff
@@ -17,6 +17,7 @@
 #   Rundeck jobs related to a project.
 #
 define rundeck::config::project (
+  Enum['absent', 'present'] $ensure = 'present',
```

Comment:
please document the parameter with puppet-strings

Reply:
I found a way